### ISSUE TITLE
How to import my own model?

### ISSUE THREAD

**Issue by fatihc07** [matterport/Mask_RCNN#2188]
Hello everyone, 
I would like to use MASK-RCNN with my own model and dataset.I already prepared my dataset and json annotation.
However, I could not import my own model and training with this model
Could you help me, please?
Here is a code example of my own model code

import keras
from keras.models import Sequential
from keras.layers import Dense, Dropout, Flatten
from keras.layers import Conv2D, MaxPooling2D

cnn1 = Sequential()
cnn1.add(Conv2D(32, kernel_size=(3, 3), activation='relu', input_shape=input_shape))
cnn1.add(MaxPooling2D(pool_size=(2, 2)))
cnn1.add(Dropout(0.2))

cnn1.add(Flatten())

cnn1.add(Dense(128, activation='relu'))
cnn1.add(Dense(10, activation='softmax'))
